src/stories/Quiz/Hooks/useCioQuiz.mdx

```diff
@@ -0,0 +1,327 @@
+import { Meta, Controls, ArgTypes, Title } from '@storybook/blocks';
+import * as HooksStories from './HooksStories.stories';
+import { Markdown } from '@storybook/blocks';
+import { BasicUsage } from '../Component/index.stories';
+import * as ComponentStories from './HooksStories.stories';
+import { e2eInteractionTest } from '../tests/index.stories';
+
+<Meta of={ComponentStories} />
+
+### <Title>useCioQuiz</Title>
+<p>Hook takes [args](#args) as a parameter and returning props giving you complete control over your UI.</p>
+
+### <h2>useCioQuiz Returns</h2>
+
+{/* -------- STATE -------- */}
+### state
+  >This is quiz state. This includes quiz answers and info.
+
+  ```jsx
+  const { state } = useCioQuiz(args);
+  ```
+  
+  <h5>Properties on <b>`state`:</b></h5>
+
+  - `answers`: Is the user entered/selected answers including:
+
+    <Markdown>
+      {`
+      | property          | type                                                          | description                                     |
+      | :---------------- | :----------------------------------                           | :-------------------------------------------    |
+      | inputs            | \`[key: string]: {type: string , value: string | string[];}\` | Is the user entered/selected answers inputs     |
+      | isLastAnswer      | Boolean                                                       | whether it's the last question                  |
+      `}
+    </Markdown>
+
+  <br />
+  - `quiz`: Quiz info including:
+
+    <Markdown>
+    {`
+    | property                      | type                                    | description                          |
+    | :-----------------------------| :---------------------------------------| :------------------------------------|
+    | currentQuestion               | Object                                  | The current question in the quiz.    |
+    | results                       | Object                                  | The quiz results                     |
+    | requestState                  | \`'STALE'|'LOADING'|'SUCCESS'|'ERROR'\` | The API request state                |
+    | versionId                     | string                                  | The quiz version                     |
+    | sessionId                     | string                                  | The quiz session                     |
+    | selectedOptionsWithAttributes | string[]                                | The selected options with attributes |
+    `}
+    </Markdown>
+
+<hr />
+
+{/* -------- EVENTS -------- */}
+### events
+> Events to call to trigger constructor events and quiz actions.
+
+  ```jsx
+  const { events } = useCioQuiz(args);
+  ```
+  Properties on <b>`events`:</b>
+
+  <Markdown>
+  {`
```

Comment:
Is it possible to put this markdown into an actual markdown file and import it from there?
The reason I'm asking is, with the current approach, it's a giant string blob. We don't get syntax highlighting and formatting. If we can put this into an `md` file and import it (which I don't know if possible), we wouldn't have to worry about formatting tables correctly

Reply:
@mocca102 sorry my comment was misleading. I meant flat `.md` files so they can be linted. Let's sync about this on a call